querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/support/RenameMapping.java

```diff
@@ -23,9 +23,9 @@
  */
 public class RenameMapping implements Mapping {
 
-    String fromSchema, fromTable, fromColumn;
+    public String fromSchema, fromTable, fromColumn;
```

Comment:
the public fields are more an intermediate form, the goal is a form that can be populated by both Maven and Ant, haven't looked into Ant population yet.